karma.headless.conf.js

```diff
@@ -0,0 +1,18 @@
+// Karma configuration
+
+module.exports = function (config) {
+    'use strict';
+
+    var commonConfig = require('./karma.common.conf');
+
+    commonConfig(config, {
+        singleRun: true,
+        autoWatch: false,
+        browsers: ['PhantomJS'],
+        logLevel: config.LOG_INFO,
+        preprocessors: {
+            'components/**/*.js': ['coverage']
+        },
+        reporters: ['progress', 'coverage']
+    });
+};
```

Comment:
I think it would be better to move this to the common conf file so we don't need a default config, headless and debug (just default (which would be headless) and debug)